src/components/animeGenres.jsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faAngleDown, faAngleRight } from '@fortawesome/free-solid-svg-icons';
+import getGenres from '../redux/animeGenres/actions';
+import Genre from './genre';
+import './animeGenres.scss';
+import { clearSlice } from '../redux/genreDetails/genreDetailsSlice';
+
+const AnimeGenres = () => {
+  const { genres, isLoading, errorMessage } = useSelector((state) => state.animeGenres);
+  const dispatch = useDispatch();
+  // This state is for this component so it's not necesary to use redux
+  const [showGenres, setShowGenres] = useState(true);
+  const [searchText, setSearchText] = useState('');
+
+  useEffect(() => {
+    dispatch(getGenres());
+    dispatch(clearSlice());
+  }, []);
+
+  const handleSearch = (e) => {
+    setSearchText(e.target.value);
+  };
+
+  const filteredGenres = genres.filter(
+    (genre) => genre.name.toLowerCase().includes(searchText.toLowerCase()),
+  );
+
+  return (
+    <section className="genres-section">
+      <div className="anime-topic">
+        <h2>Anime</h2>
+        <p>
+          (
+          {filteredGenres.length}
+          {' '}
+          genres)
+        </p>
+      </div>
+      <input type="text" placeholder="Search" className="search-bar" value={searchText} onChange={handleSearch} />
```

Comment:
- [ **OPTIONAL** ] Good adding the changes mentioned by the previous reviewer. However I think it would be better if you improve the design of `search` input in the way user can automatically know that it is a `search bar`. I can suggest you change its background color to be different to the main color, and if possible you can add there the **search** icon,... You can also change the input-type to be `search` instead of text for good semantics best practices.